docker/scripts/dev_start.sh

```diff
@@ -30,7 +30,7 @@ LOCAL_IMAGE="no"
 FAST_BUILD_MODE="no"
 FAST_TEST_MODE="no"
 VERSION=""
-VERSION_X86_64="dev-x86_64-18.04-20200714_0111"
+VERSION_X86_64="dev-x86_64-18.04-20200718_0221"
```

Comment:
Our image update frequence could be  a little slow ?

Reply:
> Our image update frequence could be a little slow ?

Just bugfix on top of previous image.